test/e2e/node_drain_test.go

```diff
@@ -97,7 +150,7 @@ func deployStatefulSetAndBlockCSI(ctx context.Context, bootstrapClusterProxy fra
 		WorkloadClusterProxy:                workloadClusterProxy,
 		ControlPlane:                        controlplane,
 		StatefulSetName:                     "sts-cp",
-		Namespace:                           "evictable-workload",
+		Namespace:                           "volume-evictable-workload",
```

Comment:
Let's add godocs here and below please why we had to pick this name (alphabetic order etc.)